libs/ui/src/components/atoms/input/input.stories.tsx

```diff
@@ -1,11 +1,57 @@
-import { ComponentMeta } from '@storybook/react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
 import Input from './input';
+import { BsSearch } from 'react-icons/bs';
 
 export default {
   title: 'Atoms/Input',
   component: Input,
 } as ComponentMeta<typeof Input>;
 
-export const InputAtom = () => <Input />;
-export const InputAtom2 = () => <Input />;
-export const InputAtom3 = () => <Input />;
+const Template: ComponentStory<typeof Input> = (args) => <Input {...args} />;
+
+export const AtomInput = Template.bind({});
+
+AtomInput.args = {
+  placeholder: 'Placeholder',
+};
+export const InputWithIcon = Template.bind({});
```

Comment:
General question -- now that you've tried it, what do you think about using the `Template.bind` approach compared to the `export const` ?

Reply:
Great. Storybook works much better with it.